src/general/WorldGenerationSettings.cs

```diff
@@ -102,15 +102,17 @@ public enum LifeOrigin
     [JsonIgnore]
     public bool FreeGlucoseCloud => Difficulty.FreeGlucoseCloud;
 
-    [JsonIgnore]
-    public bool PassiveGainOfReproductionCompounds => Difficulty.PassiveReproduction;
-
     [JsonIgnore]
     public bool SwitchSpeciesOnExtinction => Difficulty.SwitchSpeciesOnExtinction;
 
     [JsonIgnore]
     public bool LimitReproductionCompoundUseSpeed => Difficulty.LimitGrowthRate;
 
+    /// <summary>
+    ///   Whether the player can passively reproduce
+    /// </summary>
+    public bool PassiveGainOfReproductionCompounds { get; set; } = true;
```

Comment:
I think this option should also be removed, or at least this should be a JSONIgnore property that has no way to change it:
```c#
    [JsonIgnore]
    public bool PassiveGainOfReproductionCompounds => true;
```